controllers/networkplugins_controller.go

```diff
@@ -58,11 +58,11 @@ const (
 	WhereaboutsImage        = "docker.io/platform9/whereabouts:v0.6-pmk-2871011"
 	SriovCniImage           = "docker.io/platform9/sriov-cni:v2.6.2-pmk-2877848"
 	SriovDpImage            = "docker.io/platform9/sriov-network-device-plugin:v3.3.2-pmk-2877839"
-	OvsImage                = "docker.io/platform9/openvswitch:v2.17.5-1"
+	OvsImage                = "docker.io/platform9/openvswitch:v2.17.5-2"
```

Comment:
Where is this image built? What is the source of this image?

Reply:
there is no tc build for this,
we have to build this locally and push it
this is located in hostplumber/pkg/ovs_docker

Reply:
Yeah sure

Reply:
@joey00072 Do we have a JIRA issue for this now? We still see critical vulnerabilities reported in `docker.io/platform9/openvswitch`